js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,30 @@
+function getFirstAndLastWords(str) {
+  const firstWord = str.substring(0, str.indexOf(' '));
+  const lastWord = str.substring(str.lastIndexOf(' ') + 1);
+
+  if (firstWord === '') {
+    return [str, ''];
+  }
+
+  return [firstWord, lastWord];
+}
+
+function abbreviateString(str) {
+  if (typeof str !== 'string') {
```

Comment:
Check if the input is present or not like if( !str || typeof(str) === 'string') throw typeError